terraform/graph_walk_context.go

```diff
@@ -32,10 +32,10 @@ type ContextGraphWalker struct {
 	interpolaterVars    map[string]map[string]interface{}
 	interpolaterVarLock sync.Mutex
 	providerCache       map[string]ResourceProvider
-	providerConfigCache map[string]*ResourceConfig
-	providerLock        sync.Mutex
-	provisionerCache    map[string]ResourceProvisioner
-	provisionerLock     sync.Mutex
+	//providerConfigCache map[string]*ResourceConfig
```

Comment:
What's the plan for this? Will it be re-introduced by a later change you're planning?

Reply:
just missed in cleanup